docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+**Example 1**
+
+1. **"You will next open your Chrome browser. Then, open a tab."**
+2. **"We will next open our Chrome browser. Then, we will open a tab."**
+
+In the above, the #1 phrase is precise, and the reader knows what is expected. In the #2 phrase, the additional words "we will" is needed to keep the sentence relatively clear. **Note:** "Then, we open a tab." - is awkward and unclear.
+
+**Example 2**
+
+1. **"Download Node.js and install it onto your computer."**
+2. **"We download Node.js, and then we install it onto our computers."**
+
+In the above, the #1 phrase is precise. #2 is actually confusing.
+
+Therefore, when you write instructions, use "you." This guide helps to keep the documentation clearly written.
+
+---
+
+#### Ensure to use trade, project and service provider names correctly
+
+Strapi is back-end and front-end agnostic. Many different front-ends and databases can (and will) be used. Also, to front-ends and back-ends, many packages, tools, and third-party service providers are often used with Strapi. Referring to the projects and services correctly is essential. If you use incorrect terms to refer to external services, users could get confused when further research outside the Strapi docs is required.
+
+Common projects and how to refer to them:
+
+| Incorrect Usage     | Correct usage |                   Example                   |
+| ------------------- | :-----------: | :-----------------------------------------: |
+| mariadb, mariaDB    |    MariaDB    |    The power and flexibility of MariaDB.    |
+| mongo, mongodb      |    MongoDB    |     MongoDB is a document database, ...     |
+| mysql, MYSQL        |     MySQL     |  You can discuss issues with MySQL users.   |
+| Node, NODE, node.js |    Node.js    |   The Node.js Twitter account is @nodejs.   |
+| npm, NPM            |      npm      | The current stable version of npm is here.  |
+| NPX, Npx, npx       |      npx      | Run packages without downloading using npx. |
+| PM2, pm2, Pm2       |      pm2      | Guys just installed pm2 on my live server.  |
+| postgressql         |  PostgreSQL   |  Want to learn more about PostgreSQL, ...   |
+| React, react.js     |     React     | React allows you to interface with other... |
+| SqLite, sqlite      |    SQLite     |       SQLite is in the public domain.       |
+| Strapi, Strapi.js   |    Strapi     |   Get ready to get Strapi up and running.   |
+| Vue, VUE, Vue.js    | Vue or Vue.js |  My favorite component library for Vue...   |
+| yarn, YARN, Yarn    |     Yarn      | You can use Yarn to keep it all up to date. |
+
+---
+
+#### Using images
+
+If you use images in the documentation, it is a best practice to include both HTML **alt text** and **title text**. The **alt text** appears in place of a broken link, and the **title text** appears when someone hovers over the image. Both are used to help with _accessibility_.
```

Comment:
In addition you add this plugin https://www.npmjs.com/package/eslint-plugin-markdown so you won't have to pay attention to accessibility when reviewing a PR

Reply:
I invite you to read https://www.smashingmagazine.com/2018/09/importance-manual-accessibility-testing/. This article explains the dangers of what you are suggesting.

In this specific instance, alt tags and title text, the eslint plugin is never going to catch whether or not someone copy/pasted another img snippet and forgot to change the title and alt tag. IF we're serious about accessibility, all PRs will have accessibility as a key factor.  Here is a perfect example of what the eslint plugin will miss:

```
![The Strapi Logo](../assets/contribution-guide/contribute-with-docs/strapi-logo.png 'Strapi Logo')
```
```md
![The Strapi Logo](../assets/contribution-guide/restaurants/mcdonalds.png 'Strapi Logo')
```
```md
![The Strapi Logo](../assets/contribution-guide/automobiles/ferrari.png 'Ferrari Logo')
```

This is low hanging fruit to take us a little closer to actual showing how much we care about accessibility.